regression/scripts/concorde/concorde.py

```diff
@@ -66,7 +66,10 @@ def main():
     
     ## Use these scripts to test OpenVSP functionality if desired
     #from SUAVE.Input_Output.OpenVSP.vsp_write import write
```

Comment:
OpenVSP now has ubuntu binaries that supposedly support python. I wonder if now is the time to start adding them to our appveyor build? This could be a separate PR.

Reply:
I think that would be a good move if we can get it to work. I've occasionally hit broken OpenVSP code without any warning.